### ISSUE TITLE
Collision Detection is Questionable

### ISSUE THREAD

**Issue by iconjack** [jsseidel/jgalaxian#16]
Collision detection appears to be some kind of primitive bounding-rectangle type, which is very inaccurate. Not a problem on the first couple of levels, but makes the game unplayable beyond level 3.

**Comment by jsseidel** [comment]
Yes, the collision detection is a simple rectangle intersection check. And yes, this type of collision detection has an obvious flaw: if sprites move too quickly, the distance per update can be greater than the collision rect.

One simple way of fixing this is to limit the difficulty multipliers so that sprites stop getting faster at some point, like keeping the equivalent of say a level 5 speed for the remainder of the game. This is definitely worth tweaking, so thanks for the comment.

However, I routinely get to level 7. I think my best is 10. So I disagree that level 3 is the highest that can be achieved. This of course assumes that there isn't some other issue that makes the game behave differently on your particular setup.
